redhawk/common/node.py

```diff
@@ -1234,6 +1235,23 @@ def GetSExp(self):
     return li
 
 
+class Try(TryCatch):
```

Comment:
Ah, I see. The solution in that case is to have a single Try node, but to change the python_tree_convert to always emit Try.

This is important since when querying a Python AST, you will want to have the same selectors and XPATH irrespective of whether you are using Python 2 or Python 3 to parse and query.